nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -366,8 +383,8 @@ def create_model(cls, config, device):
             model_args=config.get("model_args", {}),
             init_batch_norm=config.get("init_batch_norm", False),
             device=device,
-            checkpoint_file=config.get("checkpoint_file", None),
-            load_checkpoint_args=config.get("load_checkpoint_args", {}),
+            checkpoint_file=checkpoint_file,
+            load_checkpoint_args=load_checkpoint_args,
```

Comment:
Why do you need to refactor this method passing `checkpoint_file`, `load_checkpoint_args` as arguments ignoring the values in the `config` dictionary. 
This will break resume and any other script that rely on the `config` to pass the checkpoint file.
This class method is called directly by other scripts without instantiating the experiment. Such as regression and other tests

Reply:
I kept the same signature and added an extra flag whether or not to use those arguments from config